notebooks/15_polygon_fill.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #| exporti
</pre><p>@joshuacortez suggest changing <code>exporti</code> to <code>export</code> so that <code>voxel_traversal_2d </code>is   added to <code>__all__</code> default module exports as well as showing up as documentation.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
thanks butch i addressed in the latest commit